tests/property/test_verify.py

```diff
@@ -46,7 +46,7 @@ class TestVerify(BaseTestCase):
     """Test ballot verification"""
 
     @settings(
-        deadline=timedelta(milliseconds=2000),
+        deadline=timedelta(milliseconds=5000),
```

Comment:
These changes be showing because your fork is behind master? Click the sync button on your fork to get the latest from master